src/main/resources/application.yaml

```diff
@@ -122,6 +122,11 @@ pdf:
 caseWorkerUserName: ${CASEWORKER_USER_NAME:user_name}
 caseWorkerPassword: ${CASEWORKER_PASSWORD:password}
 
+etcos:
+  system:
+    username: ${ET_COS_SYSTEM_USER:system_user}
+    password: ${ET_COS_SYSTEM_USER_PASSWORD:password}
+
```

Comment:
if you want to use the ET COS account, you will also need to add this in the Helm chart values.yaml files or you can use lines 122 and 123 as the admin user as that account already has API permissions (says caseworker but is the API user)

Reply:
Removed new system user parameters and used caseWorker ones.